src/sentry/static/sentry/app/components/events/interfaces/breadcrumbsV2/breadcrumbCategory.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import Tooltip from 'app/components/tooltip';
+import overflowEllipsis from 'app/styles/overflowEllipsis';
+
+type Props = {
+  category?: string;
+};
+
+const BreadcrumbCategory = ({category = 'generic'}: Props) => (
+  <div>
+    <Tooltip title={category}>
```

Comment:
Is this tooltip doing much if it is repeating the content that is already displayed?

Reply:
yes, the tooltip is useful, because sometimes the content is too long and we display ellipsis